tests/cases/fourslash/importNameCodeFixNewImportTypeOnly.ts

```diff
@@ -1,14 +0,0 @@
-/// <reference path="fourslash.ts" />
-
-// @target: esnext
-
-// @Filename: /a.ts
-////export interface Car {}
```

Comment:
Why was this deleted, is it because similar test already exists?

Reply:
Yeah, it was actually verifying a behavior that we actually _don’t_ want anymore (using a type-only import for new imports of types), and reversing that behavior makes it redundant with many others.